cli.py

```diff
@@ -148,9 +148,23 @@ def get(ctx, name, print_all):
 
 
 @cli.command()
-def rm():
-    """[not implemented] Remove an item from cabinet"""
-    click.echo('Sorry! not implemented yet.')
+@click.argument('name')
+@click.pass_context
+def rm(ctx, name):
+    """Remove an item from cabinet"""
+
+    click.echo('>> Removing item')
+    click.echo('Name: {0}'.format(name))
+
+    if not click.confirm('Do you want to continue?'):
+        click.echo('Action canceled')
+        return
```

Comment:
Not a requirement, just a comment, as your way seems to be more readable. But, here you could also use the method [clik.confirm](http://click.pocoo.org/5/prompts/#confirmation-prompts) like:
```python
@cli.command()
@click.argument('name')
@click.pass_context
def rm(ctx, name):
    """Remove an item from cabinet"""

    click.echo('>> Removing item')
    click.echo('Name: {0}'.format(name))
    click.confirm('Do you want to continue?', abort=True)

    account = ctx.obj.get('account')
    vault = ctx.obj.get('vault')
    password = ctx.obj.get('password')
    cab = CabinetWrapper(account, vault, password)
    cab.remove(name)
```
And it won't require the `if not click.confirm...`

Anyway, Approved! =D